### DISCUSSION TITLE
打不开WSA，Cannot open WSA after backup and restore

### DISCUSSION THREAD

**Discussion by CMJNB** [LSPosed/MagiskOnWSALocal#601] [Q&A]
我卸载普通版前，把
%LOCALAPPDATA%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\userdata.vhdx
复制到别的地方。
在安装完成后，确保root版可以打开，然后在WSA关闭的情况下，将备份复制回原来的路径，并替换。接着就打不开WSA了。
---------
Before I uninstall the normal version, put
%LOCALAPPDATA%\Packages\Microsoft CorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\userdata.vhdx
Copy it elsewhere.
After the installation is complete, make sure that the root version can be opened, and then copy the backup back to the original path and replace it when the WSA is closed. Then I can't open WSA.

**Comment by Howard20181** [comment]
版本未知
日志没有

**Reply by CMJNB** [reply]
抱歉抱歉，我第一次搞不知道该如何提供版本和日志，2304.40000.10.0这个算么

**Comment by IaemTary** [comment]
same problem, did u find a solution?

**Reply by CMJNB** [reply]
没有解决，我不知道如何提供日志，所以我已经重装了。
